api/routers/auth-router.js

```diff
@@ -0,0 +1,33 @@
+//* all authentication routes (via passport) live here - for the time being Google is being used at the only 'strategy'
+
+//* create an instance of an express router 
+const express = require('express');
+const router = express.Router(); 
+
+// base 
```

Comment:
This file will probably become obsolete and be removed... it was just a way to get my head around views and passport

Reply:
sounds good to me hope it helped. Where the views at